scripts/ps/appveyor_deploy.ps1

```diff
@@ -14,6 +14,11 @@ if ($nottesting -and $notvs2015)
         if (${env:APPVEYOR_REPO_TAG} -eq "true" -or ${env:APPVEYOR_REPO_BRANCH} -match "^develop$|^push/")
         {
             msbuild openrct2.proj /t:UploadArtifacts /logger:"C:\Program Files\AppVeyor\BuildAgent\Appveyor.MSBuildLogger.dll"
+            # PowerShell is being stupid here and hijacks calls to curl, need to call it via curl.exe explicitly
```

Comment:
Please change the comment. PowerShell is not being stupid. There is by default an alias of curl to `Invoke-WebRequest`.

Reply:
Being stupid by design is still stupid. Hijacking calls to a widely available utility that doesn't even try to mimic original's behaviour is stupid.